sklearn/svm/_classes.py

```diff
@@ -189,6 +193,22 @@ class LinearSVC(LinearClassifierMixin, SparseCoefMixin, BaseEstimator):
     [1]
     """
 
+    _parameter_constraints = {
+        "penalty": StrOptions({"l1", "l2"}),
+        "loss": StrOptions({"hinge", "squared_hinge"}),
```

Comment:
The `StrOptions` need to be part of an iterable list. Same goes for `multi_class`, and `loss` in the validation of `LinearSVR`.
```suggestion
        "penalty": [StrOptions({"l1", "l2"})],
        "loss": [StrOptions({"hinge", "squared_hinge"})],
```